lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -19,9 +19,14 @@
             <% } %>
         </span>
         <span class="sr timer-announce" aria-live="assertive"></span>
-        <span id="time_remaining_id">
-            <b>
-            </b>
+        <span class="exam-timer-clock">
+            <span id="time_remaining_id">
+                <b>
+                </b>
+            </span>
+            <button id="toggle_timer" title="<%- gettext("Hide Timer") %>" aria-label="<%- gettext("Hide Timer") %>">
```

Comment:
@cptvitamin Currently, the accessible name, title, and icon for the toggle button are all changed when it is pressed (along with the timer visibility of course). That code is in my [edx-proctoring PR](https://github.com/edx/edx-proctoring/pull/363). But it sounds like `aria-pressed=[true/false]` would be a better way to go.

This is what I plan to do:
1. Add `type=button` to the toggle button control.
2. Continue to switch `title` between `"Hide Timer"` and `"Show Timer"` (no changes there)
3. Leave `aria-label` as `"Hide Timer"` always and add an `aria-pressed` attribute that is true when the timer is hidden and false when it is shown.

Does this sound like the right approach?